shared/src/ledger/eth_bridge/storage/mod.rs

```diff
@@ -4,7 +4,16 @@ pub mod vote_tallies;
 pub mod wrapped_erc20s;
 
 use super::ADDRESS;
-use crate::types::storage::{Key, KeySeg};
+use crate::types::storage::{DbKeySeg, Key, KeySeg};
```

Comment:
Yeah I think under the #EthBridge account is an appropriate place to store these parameters. In any case they should only be modifiable via a governance vote/hard fork.